baarticle.cls

```diff
@@ -302,8 +304,10 @@
     \define@key[BA]{simple}{assignment}{\def\@basimpleassignment{#1}}
     % whether to include the blocknotice or not
     \define@boolkey[BA]{simple}{blocknotice}[true]{}
+    % display links with boxes (do not appear while printing) or as colorings (appear while printing)

```

Comment:
As a sidenote: These boxes are usally displayed by certain PDF readers and are not part of the actual PDF file.